app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewHolderFactory.kt

```diff
@@ -277,9 +277,9 @@ class CustomTransactionQueuedViewHolder(private val viewBinding: ItemTxQueuedTra
 
             dateTime.text = viewTransfer.dateTimeText
 
+            confirmationsIcon.setImageDrawable(ResourcesCompat.getDrawable(resources, viewTransfer.confirmationsIcon, theme))
             confirmations.setTextColor(ResourcesCompat.getColor(resources, viewTransfer.confirmationsTextColor, theme))
             confirmations.text = resources.getString(R.string.tx_list_confirmations, viewTransfer.confirmations, viewTransfer.threshold)
-            confirmationsIcon.visibility = View.VISIBLE
```

Comment:
Don't we risk not showing the icon, if a previous `ViewHolder` hid this icon before being recycled?

Reply:
AFAICT it is never hidden. Same for Line 252

Reply:
Can this also be removed? https://github.com/gnosis/safe-android/pull/1038/files#diff-1962f95bc4f47f31a513d4279feb183bcf252ffd6f9c519cb68e2ce468ef755aR252